### PR TITLE
CONCF-621

### PR DESCRIPTION
**Pull request by justnpT**
This test checks that scroll is enabled on modals, on certain resolutions. 
Currently there is a bug that makes the scroll disabled on the resolutions mentioned on test cases.

window sizes that should be set to:
hide Finish button 
assure scrollbar doesn't appear when the bug is present

 1. Height: Finish button has to be covered -> 590 IE / 570 Chrome, Firefox, Safari
 2. Width: IE, Safari: More than 770 / Chrome: More than 1010 / Firefox: More than 1020

The proper size to cover all browsers for the test is:
 Test Case 1:
 height: 570
 Width: 1100

Test Case 1:
 height: 570
 Width: 800

### CODE REVIEW

**Comment by hakubo** [comment]
LGTM

File: src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ModalTests.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  )
+  public void ArticleFeaturesCRUDUser_001_addModifyGallery(Dimension dimension) {
+    WikiBasePageObject base = new WikiBasePageObject(driver);
+    base.logInCookie(credentials.userName, credentials.password, wikiURL);
+    ArticlePageObject article = base.openRandomArticle(wikiURL);
+    VisualEditModePageObject visualEditMode = article.goToCurrentArticleEditPage();
+    GalleryBuilderComponentObject galleryBuilder = visualEditMode.clickGalleryButton();
+    WebElement finishButton = galleryBuilder.getFinishButton();
+    //verify Finish button is visible
+    galleryBuilder.waitForElementByElement(finishButton);
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ModalTests.java]: Assert.assertTrue(isFinishButtonVisible);

File: src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ModalTests.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    //resize window to the size that makes Finish button not visible
+    driver.manage().window().setSize(dimension);
+    //verify Finish button is not visible
+    galleryBuilder.waitForElementNotVisibleByElement(finishButton);
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ModalTests.java]: Assert.asertFalse(isFinishButtonVisible());

File: src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ModalTests.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    //scroll to Finish button
+    galleryBuilder.scrollToElement(finishButton);
+    //verify Finish button is visible
+    galleryBuilder.waitForElementByElement(finishButton);
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ModalTests.java]: Assert.assertTrue(isFinishButtonVisible());

File: src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ModalTests.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    galleryBuilder.clickFinish();
+  }
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ModalTests.java]: Add some method to gallery builder PO:
```
  public boolean isFinishButtonVisibleOnPage(){
    if(!finishButton.isDisplayed()){
      scrollToElement(finishButton);
    }
    try{
      waitForElementVisibleByElement(finishButton);
      return true;
    }catch (TimeoutException e){
      return false;
    }
  }
```
and test will look like this:

```
...
    Assert.assertTrue(galleryBuilder.isFinishButtonVisibleOnPage());
    driver.manage().window().setSize(dimension);
    Assert.assertTrue(galleryBuilder.isFinishButtonVisibleOnPage());
...
```

**Comment by vforge** [comment]
LGTM :+1:

**Pull request merged**